src/h3lib/lib/algos.c

```diff
@@ -364,8 +365,7 @@ H3Error h3NeighborRotations(H3Index origin, Direction dir, int *rotations,
 
     int newRotations = 0;
     int oldBaseCell = H3_GET_BASE_CELL(current);
-    if (oldBaseCell < 0 ||  // LCOV_EXCL_BR_LINE
-        oldBaseCell >= NUM_BASE_CELLS) {
+    if (NEVER(oldBaseCell < 0) || oldBaseCell >= NUM_BASE_CELLS) {
```

Comment:
For my info: This branch is now considered covered in our coverage metrics?

Reply:
Yes, because for coverage it just sees `0 || oldBaseCell >= NUM_BASE_CELLS` (constant folds to `oldBaseCell >= NUM_BASE_CELLS`, which is covered).